src/main/kotlin/com/cjbooms/fabrikt/generators/MutableSettings.kt

```diff
@@ -0,0 +1,20 @@
+package com.cjbooms.fabrikt.generators
+
+import com.cjbooms.fabrikt.cli.ClientCodeGenOptionType
+import com.cjbooms.fabrikt.cli.CodeGenerationType
+import com.cjbooms.fabrikt.cli.ModelCodeGenOptionType
+
+object MutableSettings {
+    var generationTypes: MutableSet<CodeGenerationType> = mutableSetOf()
+    var modelOptions: MutableSet<ModelCodeGenOptionType> = mutableSetOf()
+    var clientOptions: MutableSet<ClientCodeGenOptionType> = mutableSetOf()
+    fun updateSettings(
```

Comment:
Feels quite wrong doing this, but it makes life so much easier, and given the context it's fine

Reply:
It makes a lot of sense to me 👍